packages/qwik-city/middleware/node/http.ts

```diff
@@ -100,6 +102,13 @@ export async function fromNodeHttp(
         },
       });
     },
+    getClientInfo: () => {
+      return getClientInfo
+        ? getClientInfo(req)
+        : {
+            ip: req.socket.remoteAddress,
```

Comment:
Should this also account for x-forwarded-for?

`ip: req.headers['x-forwarded-for'] || req.socket.remoteAddress`

Reply:
No, it would be a security vulnerability, our google cloud run starter will be configured to use `x-forwarded-for`
Notice that doing that unconditionally, even when the node server is not under a reverse proxy would give the user the availity to hide their IP.

Reply:
Yes you are right, that would require a configuration the developer can set, when they run the node server behind a proxy. Only then should it use the X-Forwarded-For header.